ui/components/multichain/connected-site-popover/connected-site-popover.tsx

```diff
@@ -49,10 +53,11 @@ export const ConnectedSitePopover = forwardRef<
       isOpen={isOpen}
       style={{ width: '256px' }}
       onClickOutside={onClose}
-      offset={[0, 0]}
```

Comment:
I believe we do want some offset as this should now align with the more menu. We may need to look at the global menu to see how it's handling close on outside click as this menu doesn't seem to have the same behaviour

<img width="361" alt="Screenshot 2025-02-28 at 11 57 59 AM" src="https://github.com/user-attachments/assets/590d6a1f-c203-4528-b466-9af16ab21ad7" />

Reply:
This issue exists in all the header buttons, this is not related to this PR. Will file a separate PR to handle it.